ts/src/pro/okx.ts

```diff
@@ -1994,6 +1994,10 @@ export default class okx extends okxRest {
         }
     }
 
+    requestId () {
+        return this.milliseconds ();
```

Comment:
@sc0Vu maybe we can add some randomness here, to avoid the collisions. (rand bytes or something). this fix per se does not fix the reported behavior